ushahidi/platform-client
# adding back correct verifier-task

This pull request makes the following changes:
- Adds back verifier-task that was changed by mistake

Testing checklist:
- Run `gulp dev:verifier` from the command line
- [ ] It should start a dev server
- Navigate to /verifier
- [ ] The verifier-view should load


Fixes ushahidi/platform# .

Ping @ushahidi/platform
